NuCivic/dkan
# Fix panelizer permissions CIVIC-5807

Issue: CIVIC-5807

## Description

Site manager should not have access to the 'Customize Display` button

## How to reproduce

1. log in as a site manager
2. Click on the About link in the main menu
3. Click the Customize Display button
4. Make an edit or just click the Save button, notice that you remain on that page and do not have access to the View tab to exit the edit form.

## Acceptance Criteria
When logged in as a site manager, user should not have access to the Customize Display action tab when on a panelized page.

## Reminders
